WordPress/src/main/res/layout/scan_history_list_fragment.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.core.widget.NestedScrollView
```

Comment:
❓: I guess `NestedScrollView` is added just for demonstrating `liftOnScroll` behaviour, right?

Reply:
Yes, I plan to use recyclerView for the real content.